.gitignore

```diff
@@ -0,0 +1,3 @@
+.cache
+.vscode
+dist
```

Comment:
Add eof

Reply:
I added the line and pushed the code but it did not reflect here